modules/nodes-nodes-configuring-graceful-shutdown.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assembly:
+// * nodes/nodes-nodes-graceful-shutdown
+
+:_content-type: PROCEDURE
+[id="nodes-nodes-activating-graceful-shutdown_{context}"]
+= Configuring graceful node shutdown
+
+To configure graceful node shutdown for a node, create a `KubeletConfig` custom resource (CR) to specify a grace period and shutdown profile for specific nodes. If graceful node shutdown is not configured, the default grace period is 0 and the pod is restarted without running PreStop.
```

Comment:
Instead of "for specific nodes", maybe "for the nodes corresponding to a specific MachineConfigPool"

Reply:
Done.